packages/app/src/app/overmind/utils/auth.ts

```diff
@@ -0,0 +1,8 @@
+export type AuthOptions =
+  | {
+      provider: 'github';
+      useExtraScopes?: true; // Omit the flag if there's no need for extra scopes.
```

Comment:
What are "extra scopes" referring to, should we explain this somewhere?

Reply:
Extra scopes is referring to whether we ask for the user's email only or for the workflow and repos too. I'll document this type when I update it to be more flexible since we might want to ask for public repos only.